renv/core.py

```diff
@@ -132,49 +131,65 @@ def ensure_directories(self, env_dir):
         r_script = "Rscript"
         context.R_exe = r_exe
         context.R_script = r_script
-        context.R_version = os.path.split(self.r_path)[1]
         if self.r_bin_path:
             context.abs_R_exe = os.path.join(self.r_bin_path, r_exe)
             context.abs_R_script = os.path.join(self.r_bin_path, r_script)
         else:
             context.abs_R_exe = os.path.join(self.r_path, "bin", r_exe)
             context.abs_R_script = os.path.join(self.r_path, "bin", r_script)
         context.abs_R_path = self.r_path
-        logging.info(f"System R(version):  {self.r_path}({context.R_version})")
 
+        # Get the version of R
+        # Major Version
+        major, error = utils.system_r_call(rcmd_type="major", context=context)
+        # Minor Version
+        minor, error = utils.system_r_call(rcmd_type="major", context=context)
+
+        context.R_version = "%s.%s" % (major, minor)
+
+        logging.info("System R(version):  %s(%s)" % (self.r_path, context.R_version))
         # Begin with R-Environment R files/paths
         # Continue with system R files/paths
         if sys.platform == 'win32':  # Windows
             r_env_home = env_dir
             r_abs_home = self.r_path
             r_env_include = "include"
             r_abs_include = "include"
+            r_env_doc = "doc"
+            r_env_share = "share"
         else:  # Linux
             r_env_home = os.path.join(env_dir, 'lib', "R")
             if self.r_lib_path:
                 r_abs_home = os.path.join(self.r_lib_path, "R")
             else:
-                r_abs_home = os.path.join(self.r_path, 'lib', "R")
+                if sys.maxsize > 2**32:
+                    r_abs_home = os.path.join(self.r_path, 'lib64', "R")
+                else:
+                    r_abs_home = os.path.join(self.r_path, 'lib64', "R")
```

Comment:
the if/else resolve to the same path.... Is this intentional?

Reply:
Not at all.  The first one is asking if you're using a 64 bit operating system (_lib64_).  Everything else is 32 bit OS (_lib_).

Reply:
```suggestion
                    r_abs_home = os.path.join(self.r_path, 'lib', "R")
```